notebooks/nb.ipynb

```diff
@@ -1,51 +1,54 @@
-{

```

Comment:
<p>This is a very funny image. <em>use</em> </p><p> </p><p><strong>whatever</strong> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dr-sword/test-nb-cr/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>This image is not funny.</p>